src/TLuaInterpreter.cpp

```diff
@@ -799,6 +800,19 @@ int TLuaInterpreter::getProfileName(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getProfileTabNumber
+int TLuaInterpreter::getProfileTabNumber(lua_State* L)
+{
+    Host& host = getHostFromLua(L);
+    auto profileIndex = mudlet::self()->mpTabBar->tabIndex(host.getName());
```

Comment:
As I said before - if/when we can get reordering of profiles in the display working;  so that dragging a tab to a new position also reoders it in the main window; then the number for each profile will get rearranged to match AFAICT.

As a minimum you may want an advisory in the Wiki that the number is transitory *even currently*, This is because if a profile to the left of the one that is being considered is closed then the number of the one being considered will reduce by one as the other profile disappears...